index.js

```diff
@@ -203,9 +206,6 @@ async function pipeline(options = {}) {
       await sleep(1000);
     }
     // reload the page let icomoon read latest indexedDB data
-    await c.send('Page.reload');
```

Comment:
I'm not using icomoon-cli any more, are this operations useless now?

Reply:
Yap, we have tested and it make more issue than it solve